### ISSUE TITLE
A11y tree not working

### ISSUE THREAD

**Issue by JasonXJ** [xtermjs/xterm.js#4571]
## Details
- Browser and browser version: Chrome 114 (on linux)
- xterm.js version: 5.2.1 or ToT

### Steps to reproduce

1. Run the demo
2. Enable "screenReaderMode"
3. Check the "xterm-accessibility-tree" element in devtools. All the "listitem" inside are empty. This was working on 5.1.0.

I think we didn't actually bring back the a11y code in [here](https://github.com/xtermjs/xterm.js/pull/4340#issuecomment-1561169583)

**Comment by martinpitt** [comment]
Confirmed in https://github.com/cockpit-project/cockpit-podman/pull/1358 . We keep xterm.js at 5.1.0 for that reason.

**Comment by meganrogge** [comment]
Yeah I can see that the live region is a child before I start typing but the `rowContainer` IE `.xterm-accessibility-tree` is not ever.
https://github.com/xtermjs/xterm.js/assets/29464607/c2c67b62-072b-44cf-9b1d-2b41e3d76b5b

**Comment by meganrogge** [comment]
each time I type a char, it's hitting this
![Screenshot 2023-07-31 at 2 25 38 PM](https://github.com/xtermjs/xterm.js/assets/29464607/05100881-97f0-4647-81f5-bcb5a9e54f9e)

**Comment by meganrogge** [comment]
This'll do it - `this._rowContainer` is undefined here

![Screenshot 2023-07-31 at 2 40 04 PM](https://github.com/xtermjs/xterm.js/assets/29464607/6198051e-c910-44cf-9c80-29ba954deaa7)

**Comment by meganrogge** [comment]
maybe that's just dev tools acting weird though 🤔 because it would be throwing if that were undefined

**Comment by jerch** [comment]
Yeah `setAttribute` or access to `classList` should have thrown here, if it really were `undefined`.

Hmm its an entry in the "Watch" section, which I dont use. At least the [doc](https://developer.chrome.com/docs/devtools/javascript/watch-variables/#adding_variables) states:

> The watcher will show you the current value of the variable as it is added.
[...]
> The watch list is not a live view of the variables unless you are stepping through execution.  [...] To manually recheck the variables in the list press the refresh button to the right of the section heading.

So you prolly need to hit the refresh button to update the values again, if you jumped over a bigger execution stack.

I never had issues  with the "Scope" section thought, there should be a `this` entry under "Local" containing the proper value for `_rowContainer` right before the breakpoint state. Also these entries are updated automatically.
